packages/wrapped-keys-lit-actions/project.json

```diff
@@ -4,15 +4,15 @@
   "sourceRoot": "packages/wrapped-keys-lit-actions/src",
   "projectType": "library",
   "targets": {
-    "build-la": {
-      "cache": false,
+    "prebuild": {
       "executor": "nx:run-commands",
       "options": {
         "commands": [
           "yarn --cwd ./packages/wrapped-keys-lit-actions generate-lit-actions"
         ],
         "cwd": "."
-      }
+      },
+      "dependsOn": []
```

Comment:
The explicit dependencies definition avoids a circular dependency as both WK and WKLA depend on each other to build but on different targets

The explicit flow is as follows
`WKLA:build -> WK:build -> WK:prebuild -> WKLA:prebuild -> nothing`

Reply:
<3 nice work dude.